crates/relayer/src/link/packet_events.rs

```diff
@@ -41,14 +41,21 @@ where
             Ok(events) => {
                 events_left -= chunk.len();
 
-                info!(
-                    events.total = %events_total,
-                    events.left = %events_left,
-                    "pulled packet data for {} events out of {} sequences: {};",
-                    events.len(),
-                    chunk.len(),
-                    chunk.iter().copied().collated().format(", "),
-                );
+                if events.is_empty() && chunk.len() > events.len() {
+                    warn!("no packet data was pulled at height {query_height} for sequences {}, this might be due to the data not being available on the configured endpoint. \
+                    Please verify that the RPC endpoint has the required packet data",
```

Comment:
Can we make this more actionable for the users? It's not clear even to me what we are instructing them to do, concretely?

```suggestion
                    Please verify that the RPC endpoint has the required packet data",
```

Reply:
I added a link to the guide page, https://github.com/informalsystems/hermes/pull/4073/commits/4aefdb96e2901bad29a474f30a9bc4a1df9cbd14, let me know what you think about this.